info-leak.py

```
@@ -0,0 +1,203 @@
+        about_panel.add(about_label, gbc)
+
+        # Add the author label
+        author_label = JLabel("Author: Mark Sowell")
+        gbc.gridy = 1
+        about_panel.add(author_label, gbc)
+
+        tabbedPane.addTab("About", about_panel)
+
+        self._main_panel.add(tabbedPane)
+
+    def _import_grep_strings(self, event):
```

Ruff (0.12.2)

Unused method argument: `event`

(ARG002)